packages/web-client/app/utils/web3-strategies/sokol.ts

```diff
@@ -16,8 +16,9 @@ export default class SokolWeb3Strategy implements Layer2Web3Strategy {
   provider = new WalletConnectProvider({
     chainId: this.chainId,
     rpc: {
-      77: 'https://sokol.poa.network',
+      77: 'https://sokol.stack.cards',
     },
+    infuraId: undefined,
```

Comment:
What does `infuraId: undefined` do? I don't seem to get errors when I remove this

Reply:
Good catch. I added that while troubleshooting. Will remove.